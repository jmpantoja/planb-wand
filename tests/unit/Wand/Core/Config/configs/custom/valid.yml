---
tasks:
  init:
    actionA: true
    actionB: false
    actionC: true
