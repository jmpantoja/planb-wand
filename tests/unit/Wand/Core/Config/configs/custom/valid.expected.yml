---
tasks:
  init:
    actionA: true
    actionC: true
